src/target/riscv/riscv-013.c

```diff
@@ -1503,15 +1503,20 @@ static void deinit_target(struct target *target)
 	info->version_specific = NULL;
 }
 
-static int set_haltgroup(struct target *target, bool *supported)
+typedef enum {
+	HALTGROUP,
+	RESUMEGROUP
+} grouptype_t;
+static int set_group(struct target *target, bool *supported, unsigned group, grouptype_t grouptype)
```

Comment:
Cosmetic stuff: Blank line between the typedef and the following function.

Reply:
I intentionally didn't do that since this typedef is only relevant for this function. I really wanted to put the definition in the prototype itself, but then it's not accessible to callers.

Reply:
Ok, understood. I just made the remark because I don't recall seeing this style in other parts of OpenOCD code. I hope that folks in upstream will be OK with that.